internal/allocator/id_allocator.go

```diff
@@ -93,12 +94,7 @@ func (ia *IDAllocator) syncID() (bool, error) {
 
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	req := &rootcoordpb.AllocIDRequest{
-		Base: &commonpb.MsgBase{
-			MsgType:   commonpb.MsgType_RequestID,
-			MsgID:     0,
-			Timestamp: 0,
-			SourceID:  ia.PeerID,
-		},
+		Base:  commonpbutil.NewMsgBase(commonpb.MsgType_RequestID, 0, 0, ia.PeerID, -2),
```

Comment:
-2?

Reply:
maybe add a named Const like `TargetTSO` instead of using magic number